src/ucp/core/ucp_request.inl

```diff
@@ -788,4 +778,54 @@ ucp_request_invoke_uct_completion(ucp_request_t *req, ucs_status_t status)
     ucp_invoke_uct_completion(&req->send.state.uct_comp, status);
 }
 
+static UCS_F_ALWAYS_INLINE void
```

Comment:
rolled back
can't move `ucp_request_complete_send_common()` there, since it depends on `ucp_request_send_buffer_dereg()` that's declared in `ucp/core/ucp_request.inl`